grow/ui/sass/tool_ga_inspect.sass

```diff
@@ -0,0 +1,19 @@
+.grow__tool__ga
+  &--active
+    [data-g-event]
+      background: repeating-linear-gradient(-45deg, #FBE9E7, #FFCCBC 5px, white 5px, white 10px)
+
+      &:hover::after
+        background: #E64A19
+        border-radius: .25em
+        bottom: 50px
+        color: #ffffff
+        content: '[Event] ' attr(data-g-event) ' [Action] ' attr(data-g-action) ' [Label] ' attr(data-g-label)
```

Comment:
I have been thinking this morning that we could make the grow tools being used part of the configuration (podspec) then have them be off by default if they are not applicable to all of the grow sites. Then we can be a bit more modular with specific tools like this one and make them optional/extensible.